src/trans.ts

```diff
@@ -226,6 +226,21 @@ const trans = {
       attachments: 'Vereiste email bijlage(n)',
     },
   },
+  consultant: {
+    createNew: 'Nieuwe consultant',
+    contact: 'Contact gegevens',
```

Comment:
Wordt deze nog gebruikt?

Reply:
De 'contact' key wordt inderdaad niet meer gebruikt.

Reply:
Deze changes zullen in de nieuwe pull request zitten (feature/create_project). Dus je kan deze pull request mergen.